src/schema/organization/contributors/__tests__/expectedResults.js

```diff
@@ -0,0 +1,49 @@
+const contributors = {
+    "organization": {
+        "contributors": [
```

Comment:
I think we should start to follow a model for defining these objects

For example: 
https://github.com/panelinhadees/server/blob/master/src/schema/organization/__tests__/expectedResults.js#L3
Here, we are defining the object keys without quotation marks (""), and here you are defining with these.